spec.html

```diff
@@ -29005,17 +29028,26 @@ <h1>Runtime Semantics: Evaluation</h1>
         <emu-alg>
           1. Return the result of evaluating |VariableStatement|.
         </emu-alg>
-        <emu-grammar>ExportDeclaration : `export` Declaration</emu-grammar>
+        <emu-grammar>ExportDeclaration : DecoratorList? `export` Declaration</emu-grammar>
         <emu-alg>
-          1. Return the result of evaluating |Declaration|.
+          1. If |DecoratorList?| is present, then
+            1. Assert: |Declaration| is a |ClassDeclaration|.
+            1. Let _decorators_ be ? DecoratorListEvaluation of |DecoratorList|.
```

Comment:
Might be worth adding an assert here that |ClassDeclaration| does not have its own |DecoratorList| (and in the `export default` case as well, of course).

It's prevented by the early errors, but not by the mere structure of the nonterminals, so if you haven't noticed the early errors it seems like this is just discarding that |DecoratorList|.